### PR TITLE
Feature 300/4096 bug fix, and updates to command line + demo functionality - 2

### PR DESCRIPTION
**Pull request by mrubash1**

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1,22 @@
+my_model.hdf5 filter=lfs diff=lfs merge=lfs -text
```
Comment (hundredblocks) [inline .gitattributes]: Remove if unused?

File: README.md [hunk 2]
```diff
@@ -4,87 +4,160 @@
...
+conda create -n semantic_search python=3.5 -y
+source activate semantic_search
+pip install -r requirements.txt
+```
 
-`pip install -r requirements-all.txt`
+If you intend to use text, download pre-trained GloVe vectors (we suggest to use the length 300 vectors):
+```
+curl -LO http://nlp.stanford.edu/data/glove.6B.zip
+unzip http://nlp.stanford.edu/data/glove.6B.zip
```
Comment (hundredblocks) [inline README.md]: replace with `unzip glove.6B.zip `

File: README.md [hunk 3]
```diff
@@ -4,87 +4,160 @@
...
+mkdir models/glove.6B
+mv glove.6B.300d.txt models/glove.6B/
+```
 
-If you intend to use text, download pre-trained GloVe [vectors](http://nlp.stanford.edu/data/glove.6B.zip). Place them in `models/glove.6B/glove.6B.300d.txt`. We use the ones of length 300.
+Download an example image dataset by using:
+```
+mkdir dataset
+python downloader.py
+```
```
Comment (hundredblocks) [inline README.md:42]: add? 
```
mv dataset/diningtable dataset/dining_table
mv dataset/pottedplant dataset/potted_plant
mv dataset/tvmonitor dataset/tv_monitor
```

Comment (mrubash1) [reply README.md:42]: added!

File: README.md (hunk 2 above)
Comment (mrubash1) [reply README.md]: thanks, missed that :)

File: .gitattributes (hunk 1 above)
Comment (mrubash1) [reply .gitattributes]: yep, i just removed all of the .gitattributes for now

**Comment by mrubash1** [comment]
@hundredblocks made all changes, let me know if it is good to merge (and since I'm not an admin, you'll have to execute it)

**Comment by hundredblocks** [comment]
Done, thank you!

**Pull request merged**